frontend/www/js/omegaup/components/user/BasicInfov2.vue

```diff
@@ -127,6 +127,13 @@ export default class UserBasicInfo extends Vue {
   get urlUsername(): string {
     return `https://omegaup.com/profile/${this.profile.username}/`;
   }
+
+  get graduationDate(): string {
+    if (this.profile.graduation_date) {
+      return this.profile.graduation_date.toLocaleDateString(T.locale);
+    }
+    return '';
```

Comment:
normalmente preferimos usar el [guard clause pattern](https://refactoring.com/catalog/replaceNestedConditionalWithGuardClauses.html) donde los condicionales manejan los casos excepcionales:

```suggestion
    if (!this.profile.graduation_date) {
      return '';
    }
    return this.profile.graduation_date.toLocaleDateString(T.locale);
```